### DISCUSSION TITLE
Need help to get things figure out!!

### DISCUSSION THREAD

**Discussion by ankushT369** [AFLplusplus/AFLplusplus#2063] [General]
Hey community !!
So recently I have gone though this repo and this seems interesting to me , I am in 2nd year new to open source and big codebases like this I know some of this concepts like fuzzing , instrumentation. But I am getting confuse how to understand the structure of the code base. Can you guide me so that I can do some contributions?

**Comment by vanhauser-thc** [comment]
If you want to contribute you should start by using the software. 
once you gained some experience with it you can check the issue section for “good first issue” and try to solve that, if you are stuck then create an issue and ask.
Blindly trying to contribute is likely a dead-end
